src/main/java/jenkins/metrics/impl/VMMetricProviderImpl.java

```diff
@@ -212,49 +216,29 @@ public MetricSet getMetricSet() {
 
     private static class CpuUsageGauge extends CachedGauge<Double> {
         private final RuntimeMXBean runtimeMXBean;
-        private final Method getProcessCpuTime;
-        private final OperatingSystemMXBean operatingSystemMXBean;
+        private final com.sun.management.OperatingSystemMXBean operatingSystemMXBean;
```

Comment:
Can you please check how this behaves with openJ9 9+? I think this class is only in hotspot and hence I suspect a cnfe on other JVMs.

Reply:
I just installed a Open J9 JDK (from adapot a jdk project) and I can see the class, but I cannot find it in [eclipse/openj9](https://github.com/eclipse/openj9) repository.

Reply:
I tried to compile the project with an OpenJ9 JDK and it worked fined. Is there something else I should try?